src/main/java/com/numericalmethod/suanshu/analysis/function/polynomial/root/QuadraticRoot.java

```diff
@@ -68,6 +68,16 @@ public List<Number> solve(Polynomial polynomial) {
             roots.add(new Double(0.0));
             roots.add(new Double(-b / a));
             return roots;
+        } else if(isZero(b, 0)) {
+            if(c < 0){
+                roots.add(new Double(sqrt(-c / a)));
+                roots.add(new Double(-sqrt(-c / a)));
```

Comment:
Maybe I forgot how to do algebra, but should this be
```suggestion
                roots.add(new Double(sqrt(c / a)));
                roots.add(new Double(-sqrt(c / a)));
```